### PR TITLE
Add spreedly-iframe-browser type definitions

### PR DESCRIPTION
**Pull request by philippevezina**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/spreedly-iframe-browser/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    removeHandlers(): void;
+
+    /**
+     * Trigger tokenization of the entered credit card.
+     * On successful tokenization, the `paymentMethod` event will be triggered. On failure, the `errors` event will be triggered.
+     * @see https://docs.spreedly.com/reference/iframe/v1/#tokenizecreditcard
+     *
+     * @param additionalFields - Map of additional payment method fields to store alongside tokenized card.
+     */
+    tokenizeCreditCard(additionalFields?: spreedly.TokenizeCreditCardAdditionalFields): void;
```
Comment (peterblazejewicz) [inline types/spreedly-iframe-browser/index.d.ts:172]: is this optional really?
https://docs.spreedly.com/reference/iframe/v1/#tokenizecreditcard
also the options state `required` for some props

Comment (philippevezina) [reply types/spreedly-iframe-browser/index.d.ts]: Indeed, you are right! I have made it required. As for the required properties, it is required to either pass `full_name` or `first_name` and `last_name`. I have pushed the needed changes for that too.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@philippevezina thx!

**Comment by philippevezina** [comment]
Ready to merge

**Pull request merged**